incident_viewer_demo/src/LineChart/LineChart.jsx

```diff
@@ -275,7 +283,10 @@ class LineChart extends PureComponent {
    */
   onMouseMove(mouse) {
     // TODO: make in alphabetical order
-    const { plotAreaHeight, goodIncidentSeries, badIncidentSeries, onHighlightDate, series, xScale , xKey, yScale, yKey } = this.props;
+    const { plotAreaHeight, goodIncidentSeries, badIncidentSeries, onHighlightDate, series, xScale , xKey, yScale, hasIncident } = this.props;
```

Comment:
This would be the time to either put these in alphabetical order, or remove the TODO if you think it is not necessary... I personally have no preference and am not sure what the best coding practice is here.